docs/auto-instrumentation-install.md

```diff
@@ -90,25 +81,175 @@ kubectl get otelinst
 # splunk-otel-collector   3s    http://$(SPLUNK_OTEL_AGENT):4317
 ```
 
-</details>
-
 ### 3. Instrument application by setting an annotation
 
-An _instrumentation.opentelemetry.io/inject-{instrumentation_library}_ annotation can be added to the following:
-- Namespace: All pods within that namespace will be instrumented.
-- Pod Spec Objects: PodSpec objects that are available as part of Deployment,
-  Statefulset, or other resources can be annotated.
-- Example annotations
-  - `instrumentation.opentelemetry.io/inject-java: "true"`
-  - `instrumentation.opentelemetry.io/inject-dotnet: "true"`
-  - `instrumentation.opentelemetry.io/inject-nodejs: "true"`
-  - `instrumentation.opentelemetry.io/inject-python: "true"`
-
-The instrumentation annotations can have the following values:
-- "true" - inject and Instrumentation resource from the namespace to use.
-- "my-instrumentation" - name of Instrumentation CR instance in the current namespace to use.
-- "my-other-namespace/my-instrumentation" - name and namespace of Instrumentation CR instance in another namespace to use.
-- "false" - do not inject.
+Enable instrumentation by adding the `instrumentation.opentelemetry.io/inject-{instrumentation_library}` annotation.
+This can be applied to a namespace for all its pods or to individual PodSpec objects, available as part of
+Deployment, Statefulset, and other resources.
+
+**Annotation Values:**
+- `"true"`: Inject the `Instrumentation` resource from the namespace.
+- `"my-instrumentation"`: Use the `Instrumentation` CR instance in the current namespace.
```

Comment:
could be helpful to introduce the term since I'm not sure there's an actual "CR" shortname.

```suggestion
- `"my-instrumentation"`: Use the `Instrumentation` custom resource (CR) instance in the current namespace.
```